<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="StartPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="startLabel">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>HintonVille Library System</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="adminButton">
          <property name="font">
           <font>
            <family>Quicksand Medium</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Admin Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="staffButton">
          <property name="font">
           <font>
            <family>Quicksand Medium</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Librarian Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="patronButton">
          <property name="font">
           <font>
            <family>Quicksand Medium</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Patron Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LibrarianLoginPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="hintonTitle">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>HintonVille Library System</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="staffloginLabel">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>LIBRARIAN LOGIN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>50</number>
          </property>
          <item>
           <widget class="QLabel" name="userLabel">
            <property name="font">
             <font>
              <family>Quicksand</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Enter username:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>50</number>
          </property>
          <item>
           <widget class="QLabel" name="userLabel_2">
            <property name="font">
             <font>
              <family>Quicksand</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Enter password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>100</number>
          </property>
          <property name="horizontalSpacing">
           <number>100</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="libLoginBackButton">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="libLoginButton">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AdminLoginPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="hintonTitle_2">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>HintonVille Library System</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="staffloginLabel_2">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ADMIN LOGIN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>50</number>
          </property>
          <item>
           <widget class="QLabel" name="adminUserLabel">
            <property name="font">
             <font>
              <family>Quicksand</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Enter username:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="adminUserInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>50</number>
          </property>
          <item>
           <widget class="QLabel" name="userLabel_3">
            <property name="font">
             <font>
              <family>Quicksand</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Enter password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="adminPasswordInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>100</number>
          </property>
          <property name="horizontalSpacing">
           <number>100</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="adminLoginButton">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="adminLoginBackButton">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PatronLoginPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="hintonTitle_3">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>HintonVille Library System</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="staffloginLabel_3">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PATRON LOGIN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>70</number>
          </property>
          <item>
           <widget class="QLabel" name="libcardlabel">
            <property name="font">
             <font>
              <family>Quicksand</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Enter Library Card Number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="patronPinInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>200</number>
          </property>
          <item>
           <widget class="QLabel" name="userLabel_4">
            <property name="font">
             <font>
              <family>Quicksand</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Enter PIN:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="patronPasswordInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>100</number>
          </property>
          <property name="horizontalSpacing">
           <number>100</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="patronLoginButton">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="patronBackButton">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PatronHomePage">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="hintonTitle_4">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>HintonVille Library System</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="staffloginLabel_4">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="adminButton_2">
          <property name="font">
           <font>
            <family>Quicksand Medium</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Browse Catalogue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="adminButton_3">
          <property name="font">
           <font>
            <family>Quicksand Medium</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>My Account</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PatronCataloguePage">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="hintonTitle_5">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>HintonVille Library System</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="staffloginLabel_5">
          <property name="font">
           <font>
            <family>Quicksand Light</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Catalogue</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="CatalogueTable">
          <property name="font">
           <font>
            <family>Quicksand</family>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="autoScrollMargin">
           <number>15</number>
          </property>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
           <property name="font">
            <font>
             <family>Quicksand</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="background">
            <color>
             <red>248</red>
             <green>228</green>
             <blue>92</blue>
            </color>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Creator</string>
           </property>
           <property name="font">
            <font>
             <family>Quicksand</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="background">
            <color>
             <red>248</red>
             <green>228</green>
             <blue>92</blue>
            </color>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Format</string>
           </property>
           <property name="font">
            <font>
             <family>Quicksand</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
           <property name="background">
            <color>
             <red>248</red>
             <green>228</green>
             <blue>92</blue>
            </color>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Availability Status</string>
           </property>
           <property name="font">
            <font>
             <family>Quicksand</family>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
           <property name="background">
            <color>
             <red>248</red>
             <green>228</green>
             <blue>92</blue>
            </color>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Check Out</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Place Hold</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PatronAccountPage"/>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::MonthSection</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
